packages/dapper/src/context.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { Dapper } from "./dapper";
+
+const DapperContext = React.createContext<Dapper | null>(null);
+
+export const DapperProvider: React.FC<{ apiDomain: string }> = (props) => {
```

Comment:
Perhaps instead of taking in the `apiDomain` as a prop, we should take the `DapperInterface` instance to keep the context re-usable with other implementations too?

Reply:
Changed this as suggested.